prow/scripts/cluster-integration/helpers/generate-and-export-letsencrypt-TLS-cert.sh

```diff
@@ -38,4 +38,12 @@ docker run  --name certbot \
 TLS_CERT=$(base64 -i ./letsencrypt/live/"${DOMAIN}"/fullchain.pem | tr -d '\n')
 export TLS_CERT
 TLS_KEY=$(base64 -i ./letsencrypt/live/"${DOMAIN}"/privkey.pem   | tr -d '\n')
-export TLS_KEY
\ No newline at end of file
+export TLS_KEY
+  "${TEST_INFRA_CLUSTER_INTEGRATION_SCRIPTS}/encrypt-certs.sh"
+	#copy the cert to cloud
+	gsutil cp "./letsencrypt/live/${DOMAIN}/nightly-gke-tls-integration-app-client-cert.encrypted gs://kyma-prow-secrets/"
+    #copy the private key to cloud
+ gsutil cp "./letsencrypt/live/${DOMAIN}/nightly-gke-tls-integration-app-client-key.encrypted gs://kyma-prow-secrets/"
+
+ gsutil setmeta  -h "Cache-Control:public, max-age=60" gs://kyma-prow-secrets/nightly-gke-tls-integration-app-client-key.encrypted
```

Comment:
What is the purpose of this cache in this case? It will not remove objects older than 60 days from GCP. It just set how long browser should cache object.

So what if the certificate expires? I don't see in this PR a logic responsible for verification if the certificate is still valid and for generating a new one in such case.